sirepo/template/raydata.py

```diff
@@ -4,6 +4,7 @@
 :copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
+from glob import glob
```

Comment:
We try to follow the google style guide it [says this](https://google.github.io/styleguide/pyguide.html#22-imports) about imports
`Use import statements for packages and modules only, not for individual classes or functions`. [glob](https://github.com/python/cpython/blob/ae36cd1e792db9d6db4c6847ec2a7d50a71f2b68/Lib/glob.py#L13) is a function.

So just `import glob` and then `glob.glob()` when you call it.

It seems pretty redundant in this case because the module and function have the same name. But for cases where they don't it helps reduce confusion. For example, we have `sirepo.tornado.Event` which is a subclass of `tornado.locks.Event`. If we did `from tornado.locks import Event` it would be confusing which "Event" we were talking about.